### PR TITLE
Bump Nessie to 0.54.0

### PR DESCRIPTION
**Pull request by snazy**
Includes #7061

### CODE REVIEW

**Review by dimas-b — APPROVED** [review]

**Comment by dimas-b** [comment]
@Fokko : Could you merge this, please?

File: versions.props [hunk 1]
```diff
@@ -22,7 +22,7 @@ javax.xml.bind:jaxb-api = 2.3.1
 javax.activation:activation = 1.1.1
 org.glassfish.jaxb:jaxb-runtime = 2.3.3
 software.amazon.awssdk:* = 2.20.18
-org.projectnessie.nessie:* = 0.51.1
+org.projectnessie.nessie:* = 0.52.3
```
Comment (ajantha-bhat) [inline versions.props:25]: maybe 0.53.0 now?

**Comment by ajantha-bhat** [comment]
cc: @nastra, @Fokko

**Review by dimas-b — APPROVED** [review]

**Review by dimas-b — APPROVED** [review]

**Comment by dimas-b** [comment]
Thanks for fixing the commit message, @Fokko ! :)

**Pull request merged**